spec/datadog/profiling/integration_spec.rb

```diff
@@ -255,9 +255,8 @@ def stack_frame_to_function_id(backtrace_location)
           is_expected.to have(4).items
 
           # All but last are unique
-          (0..-2).each do |i|
+          3.times do |i|
```

Comment:
@ivoanjo wanted to bring this to your attention. `(0..-2).each` never executes.

Reply:
Oh wow, thanks for spotting it! The good news is -- this test is only for the old profiler codepath, that is off by default and no customers should be using anymore.

The new profiler has similar specs, but in different files, and doesn't have this bug as far as I can spot it.

(The old profiler, along with this file are going to be deleted in Q4; I've mostly been finishing up a few other features before I come and remove all of this.)